docs/_getting-started/dev-mode.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Dev mode
+permalink: docs/dev-mode
```

Comment:
What does this change do? Does it change the URL of the page?

Reply:
yea - to bring it in line with all other pages and prevent some 404s that exist currently

Reply:
At the moment, the URL of this page is https://geth.ethereum.org/getting-started/dev-mode. If there are any links to `docs/dev-mode`, we should update them to `getting-started/dev-mode` instead of changing the URL.

Reply:
Not really sure about this - I think it makes more sense to bring it in line with all other pages. All other pages reside in docs/ - it was just forgotten for these pages as far as I see ..

Reply:
Oh wait, I see it now! Sorry!

I still think we shouldn't just change the URL of the page. Maybe there is a way to create redirects so we can at least keep the old URL alive?

Reply:
Unfortunately I found no way to do it without a plugin. Redirect via plugin here: f647026c3ace5f35a14e77da5735c343a43b4e00
Locally it works - hope github is also happy with it